plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -250,9 +251,12 @@ class NewServiceFormModal extends Component {
   getModalContent() {
     // NOTE: Always prioritize review screen check
     if (this.state.serviceReviewActive) {
+      // return (
```

Comment:
I guess we shouldn't remove the ServiceConfigDisplay all together? Am I missing something?